banquet/views.py

```diff
@@ -870,31 +872,41 @@ def external_banquet_afterparty(request, token=None):
 
         if request.POST and form.is_valid():
             ticket = form.save()
-            return redirect('banquet_external_afterparty_token', ticket.token)
+            ticket.save()
+            #return redirect('banquet_external_afterparty_token', ticket.token)
 
-    elif request.POST and ticket.paid_price is None:
-        stripe.api_key = settings.STRIPE_SECRET
+        if request.POST and ticket.paid_price is None:
```

Comment:
Detta vill vi väl bara ska göras om form.is_valid()? Så att man inte redirectas utan att ha fyllt i varken namn eller email. Allt med payment intents skulle ju kunna läggas under samma "if request.POST and form.is_valid():" som ligger precis ovanför? Att bara lägga "if ticket.paid_price is None" där med. Eller har jag missat nåt?